bessctl/commands.py

```diff
@@ -476,6 +477,13 @@ def bind_var(cli, var_type, line):
                 raise cli.BindError('"wid" must be a positive number')
         val = sorted(list(set(val)))
 
+    elif var_type == 'host':
+        dns = re.match(r'^[a-zA-Z0-9][a-zA-Z0-9\-.]*$', val)
```

Comment:
This is overly forgiving, e.g., it accepts `host.dom..ain`, but I don't think a full-name-compliance-checker is warranted here unless there's one handy in a library (is there?). Presumably we don't care to allow Unicode IDN input 😺

(pointless thoroughness: there are some sample Python validators in https://stackoverflow.com/questions/2532053/validate-a-hostname-string)